### ISSUE TITLE
bug: /bin/bash: no such file or directory

### ISSUE THREAD

**Issue by apawlik** [stencila/dockter#54]
Running build results in an error: 
```
 Error: (HTTP code 400) unexpected - OCI runtime create failed: container_linux.go:348: starting container process caused "exec: \"/bin/bash\": stat /bin/bash: no such file or directory": unknown
```
It is probably because the location of `bash` in the container is in `/usr/local/bin/bash`
[see here](https://docs.docker.com/samples/library/bash/)

Looks like the solution is to replace
```
RUN useradd --create-home --uid 1001 -s /bin/bash dockteruser
```
with

```
RUN useradd --create-home --uid 1001 -s  /usr/local/bin/bash dockteruser
```

@nokome Can you have a look? I can change that and push.

**Comment by apawlik** [comment]
Well, maybe not. This is not the solution.

**Comment by nokome** [comment]
> It is probably because the location of bash in the container is in /usr/local/bin/bash

I don't think that is the issue. The article you link to is for the `bash` image (ie. an image with only bash in it). On Ubuntu (which is the base image we use most of the time)...

```
nokome@pluto ~> docker run -it ubuntu:18.04 bash
root@0c925540ba9d:/# ls /bin/bash -lt
-rwxr-xr-x 1 root root 1113504 Apr  4  2018 /bin/bash
root@0c925540ba9d:/# 
```

Can you give me a few more details, the generated `.Dockerfile` would probably be most useful.

**Comment by apawlik** [comment]
```
# Generated by Dockter 0.10.3 at 2018-11-05T04:53:42.998Z
# To stop Dockter generating this file and start editing it yourself,
# rename it to "Dockerfile".

# This tells Docker which base image to use.
FROM ubuntu:16.04

# This section sets environment variables within the image.
ENV TZ="Etc/UTC" \
    R_LIBS_USER="~/R"

# This section installs system packages needed to add extra system repositories.
RUN apt-get update \
 && DEBIAN_FRONTEND=noninteractive apt-get install -y \
      apt-transport-https \
      ca-certificates \
      curl \
      software-properties-common

# This section adds system repositories required to install extra system packages.
RUN apt-add-repository "deb https://mran.microsoft.com/snapshot/2018-11-04/bin/linux/ubuntu xenial/"
RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 51716619E084DAB9

# This section installs system packages required for your project
# If you need extra system packages add them here.
RUN apt-get update \
 && DEBIAN_FRONTEND=noninteractive apt-get install -y \
      r-base \
 && apt-get autoremove -y \
 && apt-get clean \
 && rm -rf /var/lib/apt/lists/*

# This section runs commands to install Stencila execution hosts.
RUN apt-get update \
 && apt-get install -y  zlib1g-dev libxml2-dev pkg-config \
 && apt-get autoremove -y \
 && apt-get clean \
 && rm -rf /var/lib/apt/lists/* \
 && Rscript -e 'install.packages("devtools")' \
 && Rscript -e 'source("https://bioconductor.org/biocLite.R"); biocLite("graph")' \
 && Rscript -e 'devtools::install_github("r-lib/pkgbuild")' \
 && Rscript -e 'devtools::install_github("stencila/r")'

# It's good practice to run Docker images as a non-root user.
# This section creates a new user, sets it as the user for the image, and it's
# home directory as the working directory.
RUN useradd --create-home --uid 1001 -s /usr/local/bin/bash dockteruser
USER dockteruser
WORKDIR /home/dockteruser

# This is a special comment to tell Dockter to manage the build from here on
# dockter

# This section copies package requirement files into the image
COPY .DESCRIPTION DESCRIPTION

# This section runs commands to install the packages specified in the requirement file/s
RUN mkdir ~/R \
 && bash -c "Rscript <(curl -sL https://unpkg.com/@stencila/dockter/src/install.R)"

```

**Comment by nokome** [comment]
Fixed in 96a84860c7933b93c02c3044f8426673367b3e85
